coderd/database/queries/users.sql

```diff
@@ -1,3 +1,11 @@
+-- name: UpdateUserLoginType :one
+UPDATE
+	users
+SET
+	login_type = @login_type
+WHERE
+	id = @user_id RETURNING *;
```

Comment:
This should clear the stored hashed password too

Reply:
Done and added a unit test.